src/main/scala/org/ergoplatform/nodeView/wallet/persistence/WalletDigest.scala

```diff
@@ -18,7 +20,44 @@ import scorex.util.Extensions._
   */
 final case class WalletDigest(height: Int,
                               walletBalance: Long,
-                              walletAssetBalances: mutable.LinkedHashMap[EncodedTokenId, Long])
+                              walletAssetBalances: mutable.LinkedHashMap[EncodedTokenId, Long]) {
```

Comment:
let's make this mutable map private, otherwise it is error prone.
Or even better to replace it with immutable data structure.

Reply:
done, made immutable